arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -715,6 +795,64 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/workflows/{workflowName}/versions/{versionId}/triggers/{triggerName}/listCallbackUrl": {
+      "post": {
```

Comment:
I'm still not convinced about calling it `list*` . I'd like to get the opinion of @ravbhatnagar and @salameer here.